docs/standard/io/buffers.md

```diff
@@ -0,0 +1,188 @@
+---
+title: "System.Buffers - .NET"
+ms.date: "10/15/2019"
+ms.technology: dotnet-standard
+helpviewer_keywords: 
+  - "buffers [.NET]"
+  - "I/O [.NET], buffers"
+author: rick-anderson
+ms.author: riande
+---
+# Work with Buffers in .NET
+
+This article gives you an overview of types that help you read data that runs across multiple buffers. They're primarily used to support <xref:System.IO.Pipelines.PipeReader> objects.
```

Comment:
Would it be better to cross link to the pipelines article here instead of the type?